docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -105,105 +109,115 @@ significant changes in each minor release.
 
-Changelogs
-^^^^^^^^^^^^
+If there are major problems in RC2, the cycle begins again with another release candidate and repeats until the maintainers agree that all major problems have been fixed.
 
-We generate changelogs based on fragments. Here is the generated changelog for 2.9_ as an example. When creating new features or fixing bugs, create a changelog fragment describing the change. A changelog entry is not needed for new modules or plugins. Details for those items will be generated from the module documentation.
 
-We've got :ref:`examples and instructions on creating changelog fragments <changelogs_how_to>` in the Community Guide.
+.. _development_and_stable_version_maintenance_workflow:
 
+Development and maintenance workflows
+=====================================
 
-Release candidates
-^^^^^^^^^^^^^^^^^^^
+In between releases, the Ansible community develops new features, maintains existing functionality, and fixes bugs in ``ansible-core`` and in the collections included in the Ansible community package.
 
-Before a new release or version of Ansible or ``ansible-core`` can be
-done, it will typically go through a release candidate process.
+Workflow for the Ansible community package
+------------------------------------------
 
-This provides the Ansible community the opportunity to test these releases and report
-bugs or issues they might come across.
+The Ansible community develops and maintains the features and functionality included in the Ansible community package in Collections repositories, with a workflow that looks like this:
 
-Ansible and ``ansible-core`` tag the first release candidate (``RC1``)
-which is usually scheduled to last five business days. The final release
-is done if no major bugs or issues are identified during this period.
+ * Developers add new features and bug fixes to the individual Collections, following each Collection's rules on contributing.
+ * Each new feature and each bug fix includes a changelog fragment describing the work.
+ * Release engineers create a minor release for the current version every three weeks to ensure that the latest bug fixes are available to users.
+ * At the end of the development period, the release engineers announce which Collections, and which major version of each included Collection,  will be included in the next release of the Ansible community package. New Collections and new major versions may not be added after this, and the work of creating a new release begins.
 
-If there are major problems with the first candidate, a second candidate will
-be tagged (``RC2``) once the necessary fixes have landed.
-This second candidate lasts for a shorter duration than the first.
-If no problems have been reported after two business days, the final release is
-done.
+We generally do not provide fixes for unmaintained releases of the Ansible community package, however, there can sometimes be exceptions for critical issues.
```

Comment:
Suggestion: `However, there are times when exceptions are made for critical issues.`